var-test/src/test/java/io/varhttp/LauncherTestBase.java

```diff
@@ -433,4 +434,21 @@ public void noSerializerCustomContentType_unhappy() throws Throwable {
 				.getContent();
 		assertEquals("io.varhttp.ContentTypeException Requested Content-Type 'my/xml' is not supported", response);
 	}
+
+
+	@Test
+	public void streamedOut_thenFailed() {
+		varClient.get("/streamed-then-failed")
+				.execute()
+				.isInternalError()
+				.assertContent("tadaaajava.lang.RuntimeException should result in error code 500");
```

Comment:
arghh, this looks so bad. Let's make sure we write a story to not have this kind of behaviour.

Reply:
https://persequor.atlassian.net/browse/SAGA-7392